web/pdf_page_view.js

```diff
@@ -245,6 +247,7 @@ class PDFPageView {
       scale: this.scale * PixelsPerInch.PDF_TO_CSS_UNITS,
       rotation: totalRotation,
     });
+    this.#setDimensions();
```

Comment:
Do you perhaps also want to set `this.#previousRotation = totalRotation;` here, to ensure that the value is always correct?
It seems to me that that could *potentially* avoid having to re-run `#setDimensions` is some more cases in the `update`-method.

*Ninja-edit:* Or possible just add something like `this.#previousRotation = viewport.rotation;` at the top of the `#setDimensions`-method, to avoid having to do that *manually* in multiple spots?